modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -18,7 +18,7 @@
 :component: prometheus
 // end::UWM[]
 
-You can create cluster ID labels for metrics by adding the `write_relabel` settings for remote write storage in the `{configmap-name}` config map in the `{namespace-name}` namespace.
+You can create cluster ID labels for metrics by adding the `write_relabel` settings for remote write storage in the `{configmap-name}` config map in the `{namespace-name}` namespace. Adding a cluster ID label ensures that metrics can be uniquely identified and consistently tracked across clusters and workloads.
```

Comment:
"can be uniquely identified" and "consistently tracked across clusters" are passive voice.